src/playground/addons/story-utils/index.js

```diff
@@ -539,95 +539,113 @@ export const correctSvgPath = (data) => {
+      defaultValue: { summary: '' },
+      category: 'Content',
     },
-    required_text: {
-      name: 'required text',
-      type: 'string',
-      defaultValue: data.required_text,
-      description: 'Text to be shown when the form element is mandatory',
-      table: {
-        type: { summary: 'string' },
-        defaultValue: { summary: '*' },
-        category: 'Content',
-      },
+  };
+  argTypes.required_text = {
+    name: 'required text',
+    type: 'string',
+    defaultValue: data.required_text,
+    description: `Text to be shown when the form ${type} is mandatory`,
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '*' },
+      category: 'Content',
     },
-    invalid: {
-      name: 'invalid',
-      type: 'boolean',
-      defaultValue: data.invalid,
-      description: 'Marks the form element as invalid',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-      },
+  };
+  argTypes.invalid = {
+    name: 'invalid',
+    type: 'boolean',
+    defaultValue: data.invalid,
+    description: `Marks the form ${type} as invalid`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
     },
-    disabled: {
-      name: 'disabled',
-      type: 'boolean',
-      defaultValue: data.disabled,
-      description: 'Disabled form element',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-        disable: type !== 'element',
-      },
+  };
+  argTypes.disabled = {
+    name: 'disabled',
+    type: 'boolean',
+    defaultValue: data.disabled,
+    description: `Disabled form ${type}`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
+      disable: type !== 'element',
     },
-    required: {
-      name: 'required',
-      type: 'boolean',
-      defaultValue: data.required,
-      description: 'Sets the required attribute on the form element',
-      table: {
-        type: { summary: 'boolean' },
-        defaultValue: { summary: false },
-        category: 'States',
-      },
+  };
+  argTypes.required = {
+    name: 'required',
+    type: 'boolean',
+    defaultValue: data.required,
+    description: `Sets the required attribute on the form ${type}`,
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
     },
   };
+
+  if (data.width) {
+    argTypes.width = {
+      name: 'width',
+      type: { name: 'select' },
+      defaultValue: data.width,
+      description: `The width of the form ${type} {s: small, m: medium, l: large}`,
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: data.width },
+        category: 'Size',
+      },
+      control: {
+        type: 'select',
+        options: ['s', 'm', 'l'],
```

Comment:
options should be consistent with https://github.com/ec-europa/europa-component-library/pull/1842/files#diff-a7602161f00bf48ed6e08f933df9b586729725e3ddc921d3f84a95832f7a01c3R645